day-zero/live-service-templates.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Live Service Templates'
```

Comment:
```suggestion
title: 'Live Templates'
```

following the recent changes - templates can now be used for services and blueprints